features/step_definitions/branch_steps.rb

```diff
@@ -17,7 +17,7 @@
   create_branch branch_name, remote: !local
   if relation
     commit_to_branch = relation == 'behind' ? 'main' : branch_name
-    create_commits branch: commit_to_branch
+    create_commits branch: commit_to_branch, message: "#{relation} #{branch_name} commit"
```

Comment:
Good point. I have reverted this commit message, and set up the commit manually in the one spec that uses this.